app/app.js

```diff
@@ -63,6 +63,29 @@ function (app, $, _, Backbone, Bootstrap, Helpers, Utils, FauxtonAPI, Couchdb) {
   // Localize or create a new JavaScript Template object
   var JST = window.JST = window.JST || {};
 
+  var parseCookies = function (cookies) {
```

Comment:
You can simplify this function by using [`_.reduce`](http://underscorejs.org/#reduce).